lambda/get_issues/get_issues.py

```diff
@@ -0,0 +1,13 @@
+import parser
+from apsi_database.database import get_db_session
+from apsi_database.models import Report
+import apsi_database.insert_example
+
+def get_issues():
+  """
+  Gets all reports from database.
+  """
+  with get_db_session() as session:
+    records = session.query(Report).all()
+  records = [parser.object_as_dict(record) for record in records]
```

Comment:
ten parser to wewnętrzna biblioteka pythonowa? Czy może nie została dodana, bo nie widzę zmian w `requirements` lub nowych plików

Reply:
Dodałam odpowiedni plik po poprawkach. Dodałam także layer do lambdy i uzupełniłam plik requirements.txt o sqlalchemy.

Reply:
w sumie to wydaje mi się, że SQLAlchemy w `requirements` jest redundantne bo to już jest w layerze dodane, czy to jednak jest wymagane?

Reply:
Racja, nie jest potrzebne. Usunęłam.